landa/water_body_management/doctype/water_body_management_local_organization/water_body_management_local_organization_list.js

```diff
@@ -0,0 +1,13 @@
+frappe.listview_settings['Water Body Management Local Organization'] = {
+	onload: function(listview) {
+		if (listview.page.fields_dict.regional_organization) {
+			listview.page.fields_dict.regional_organization.get_query = function() {
+				return {
+					"filters": {
+						"name": ["in", ["AVL", "AVS", "AVE"]],
```

Comment:
```suggestion
						"parent": "LV",
```

Does this work, as a shorter and slightly more flexible alternative?